src/core/components/tabs/tabsContent/tabsContent.tsx

```diff
@@ -1,12 +1,25 @@
-import { TabsContent as RadixTabsContent, type TabsContentProps as RadixTabsContentProps } from '@radix-ui/react-tabs';
+import { TabsContent as RadixTabsContent } from '@radix-ui/react-tabs';
+import { type ComponentProps } from 'react';
 
-export interface ITabsContentProps extends RadixTabsContentProps {
+export interface ITabsContentProps extends ComponentProps<'div'> {
     /**
      * Value linking Tabs.Content to its corresponding Tabs.Trigger
      */
     value: string;
+    /**
+     * When `true`, the content will stay mounted even when inactive.
+     */
+    forceMount?: true;
 }
 
+/**
+ * `TabsContent` component
+ *
+ * This component is based on the Radix-UI tabs content implementation.
+ * An exhaustive list of its properties can be found in the corresponding Radix primitive
+ * [documentation](https://www.radix-ui.com/primitives/docs/components/tabs#content).
+ */
 export const TabsContent: React.FC<ITabsContentProps> = (props) => {
-    return <RadixTabsContent {...props} />;
+    const { value, forceMount, ...rest } = props;
+    return <RadixTabsContent value={value} forceMount={forceMount} {...rest} />;
```

Comment:
No need to destructure `value` and `forceMount` props, right?

Reply:
✅ https://github.com/aragon/ods/pull/218/commits/4c17b0b75d8702e9373c51aadd4f20286f81c89f